.github/workflows/sphinx.yml

```diff
@@ -0,0 +1,18 @@
+name: Sphinx Pages
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    name: Sphinx Pages
+    runs-on: ubuntu-latest
+    steps:
+      - uses: seanzhengw/sphinx-pages@master
```

Comment:
This needs a check with the security team, might not be great to run external code that can be changed at any point in time without notification, with access to at least the github token.

Reply:
You're right. The easy way out would be to fork this under an Aiven owned repo to make sure it doesn't change.

Reply:
Can't we simply specfy @commitid instead of @master ?

Reply:
Very good idea ! I committed that. Do we still need a security team check ?

Reply:
The code from `seanzhengw/sphinx-pages` code does not seem to do anything bad.

If we're being pedantic the next part of the supply chain that is not pinned is the `sphinxdoc/sphinx` docker image and the `sphinx_rtd_theme` pip package but it's not the case for this repo either so I'd say it's good enough.